README.md

```diff
@@ -179,7 +179,7 @@ environment variable, which in turn defaults to the `.ort` directory below the c
 
```

Comment:
Commit message nit: The title is a bit strange. Maybe "Move the reference configuration *to* a resource" or "*Make* the reference configuration a resource" ?

Reply:
It was not just a bit strange, but just completely broken grammar 😄  I guess I could not decide between the two when I wrote this. I've added a "to" now 👍🏻